eda-env/Lib/site-packages/pandas/util/version/__init__.py

```
@@ -0,0 +1,579 @@
+# This file is dual licensed under the terms of the Apache License, Version
+# 2.0, and the BSD License. Licence at LICENSES/PACKAGING_LICENSE
+from __future__ import annotations
+
+import collections
+from collections.abc import Iterator
+import itertools
+import re
+from typing import (
+    Callable,
+    SupportsInt,
+    Tuple,
```

Ruff (0.11.9)

`typing.Tuple` imported but unused

(F401)